corehq/apps/registration/models.py

```diff
@@ -42,6 +42,27 @@ def get_requests_today(cls):
         return result[0]['value']
 
     @classmethod
+    def get_requests_24hrs_ago(cls):
+        today = datetime.datetime.utcnow()
+        yesterday = today - datetime.timedelta(1)
+        join_on_start = datetime.datetime(
+            yesterday.year, yesterday.month, yesterday.day, yesterday.hour, 0, 0, 0)
+        join_on_end = datetime.datetime(
+            yesterday.year, yesterday.month, yesterday.day, yesterday.hour, 59, 59, 59)
```

Comment:
Nitpicky, but you might just leave off the microseconds or else use 999999 instead of 59